### DISCUSSION TITLE
ZepetoId 아이디 가져오기.

### DISCUSSION THREAD

**Discussion by Obsihill** [naverz/zepeto-studio-kor#1248] [5. Q&A]
클라이언트상에서 아래 플레이어 값 이외 로컬플레이어 자신의 아이디 값을 가져오는방법은 없나요?
sessionId
userId
hashCode

**Comment by Obsihill** [comment]
아래 방법으로 해결했습니다.
ZepetoWorldHelper.GetUserInfo

**Reply by Zepeto-TechEvangelist** [reply]
넵, 아래 가이드에서 상세 내용을 확인하실 수 있습니다. https://studio.zepeto.me/guides/searching-userinformation
